apps/omg_watcher_info/lib/omg_watcher_info/db/txoutput.ex

```diff
@@ -134,7 +135,7 @@ defmodule OMG.WatcherInfo.DB.TxOutput do
         select: {txoutput.currency, sum(txoutput.amount)}
       )
 
-    DB.Repo.all(query)
+    Repo.all(query)
     |> Enum.map(fn {currency, amount} ->
```

Comment:
this was a good oppurtunity to avoid a single pipe
query
|> Repo.all()
|> Enum.map(fn {currency, amount} ->
...

Reply:
@InoMurko yes, I thought about it. But I decided now to include additional changes. Also, there is a credo check for this stuff.

Reply:
yeah, we can't use it because we have too many of them :D

so we do this:
https://github.com/omgnetwork/elixir-omg/blob/master/.circleci/config.yml#L181-L192